src/components/CustomerLogos.tsx

```diff
@@ -1,99 +1,208 @@
-                        src={logo.src}
-                        alt={`${logo.name} logo`}
-                        title={`${logo.name} logo`}
-                        className="w-100"
-                        draggable={false}
-                    />
-                </a>
-            </Link>
-        ))}
-    </div>
-)
+/**
+ * This is our Logo Pond from the DLS which displays customer logos.
+ *
+ * @param props - component props
+ * @param props.overline - optional overline heading
+ * @param props.headline - optional headline
+ * @param props.description - optional description
+ * @param props.dark - dark mode
+ * @param props.monochrome - monochrome
+ *
+ */
+export const CustomerLogos: FunctionComponent<CustomerLogos> = ({
+    overline,
+    headline,
+    description,
+    dark,
+    monochrome,
+}) => {
+    const container = useRef<HTMLDivElement>(null)
+    const [containerWidth, setContainerWidth] = useState<number>(0)
+
+    /**
+     * This allows us to check for the logo container width so we can
+     * add a bottom margin to only the first row of logos on desktop.
+     */
+    useEffect(() => {
+        const offsetWidth = container.current?.offsetWidth
+
+        const resizeListener = (): void => {
+            if (offsetWidth) {
+                setContainerWidth(offsetWidth)
+            }
+        }
+
+        resizeListener()
+
+        window.addEventListener('resize', resizeListener)
+
+        return () => {
+            window.removeEventListener('resize', resizeListener)
+        }
+    }, [])
+
+    return (
+        <div className={classNames({ 'tw-bg-black tw-text-white': dark })}>
+            <div className="tw-mb-xl tw-text-center tw-max-w-xl tw-mx-auto">
+                {overline && (
+                    <Heading size="h6" as="h2">
+                        {overline}
+                    </Heading>
+                )}
+                {headline && (
+                    <Heading size="h2" as="h3" className="tw-my-2">
+                        {headline}
+                    </Heading>
+                )}
+                {description && <p className="tw-text-lg">{description}</p>}
+            </div>
+
+            <div
+                className={classNames(
+                    'tw-flex-wrap tw-mx-auto tw-flex tw-items-center tw-justify-center tw-max-w-screen-xl tw-select-none',
+                    { 'tw-brightness-0': dark || monochrome, 'tw-invert': dark }
+                )}
+                ref={container}
+            >
+                {logos.map((logo: Logo, index) => (
+                    <Link key={logo.name} href={logo.link ? logo.link : '/case-studies'} passHref={true}>
+                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                        <a
+                            className={classNames('tw-mx-7 tw-mb-xl tw-shrink-0', {
+                                'xl:tw-mb-0': index > (logos.length - 1) / 2 && containerWidth >= breakpoints.xl,
```

Comment:
It's only applied when the container is smaller than `1536px` (xl), not the viewport. So if we include the CustomerLogos component with a container in a layout/design that's smaller than `1536px`, then it will handle this line accordingly. We need the `xl:` modifier as well since we're overriding `tw-mb-xl` depending if the condition is met

Reply:
Is it being used anywhere now? I can't find it applied. Not in the contact pages which should have fitting criteria, no?

Reply:
Hmm.. let me take a look again. It was working on Friday 🤔

Reply:
Found the culprit! This must have been working before I changed our breakpoints to reference our Tailwind breakpoints. 😄  The values included the unit of measurement from the config, so they were defined as strings instead of numbers. Now they're numbers which is what we're expecting, so we can reference them in JS. Thanks for catching!

Reply:
Cool! I still don't see it applied hah, which section can I test this on?

Reply:
Check out the `CustomerLogos` section on the homepage and on a contact page. The margin should only be applied to the second row (half the logos) when the parent container is less than xl. The CustomerLogos on contact pages should have the margin applied since the parent container is a lot smaller for that layout. [See loom](https://www.loom.com/share/7a3c06509c5b4fbe899af80161f9db9e)

Reply:
Appreciate it! Cool, looks groovy 😎